src/main/java/core/basesyntax/reportservices/ReportCreatorImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.reportservices;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportCreatorImpl implements ReportCreator {
```

Comment:
There is an unnecessary empty line at the start of the class declaration. According to the checklist, you should remove all redundant empty lines.